splunk/src/main/java/com/splunk/HttpService.java

```diff
@@ -527,41 +535,43 @@ public void checkServerTrusted(X509Certificate[] certs, String authType) {
                 }
         };
         try {
-            SSLContext context;
-            switch (HttpService.sslSecurityProtocol) {
-                case TLSv1_2:
-                case TLSv1_1:
-                case TLSv1:
-                    context = SSLContext.getInstance("TLS");
-                    break;
-                default:
-                    context = SSLContext.getInstance("SSL");
+            String contextStr = "";
+            if (sslSecurityProtocol != null) {
+                contextStr = sslSecurityProtocol.toString().contains("SSL") ? "SSL" : "TLS";

```

Comment:
- [ ] Let's capture changing the default behavior to use TLS and turning on hostname verification for all hosts other than localhost in the RELEASE_NOTES.md since it is developer facing